scripts/package/fpsci_packager.py

```diff
@@ -0,0 +1,68 @@
+# This script generates a packaging script based on the files used by the last successful run of the program.
+# You need to run the generated script to create the distribution directory, then zip that up to send to people.
+
+# Name of the packaging script to generate
+outputScript = 'package.sh'
+
+# Path of the log.txt file to use
+inputLog = 'data-files/log.txt'
+
+# The list of files that should always be excluded
+excludeList = ['log.txt', 'keymap.Any', 'systemconfig.Any']
+# The list of files that should be excluded for release builds, but may want to be included for specific experiment distributions
+genericExcludeList = ['experimentconfig.Any', 'startupconfig.Any', 'userconfig.Any', 'userstatus.Any']
```

Comment:
Could use a more descriptive name here, something like `experimentSpecificExcludeList`, but this is largely preference.

Reply:
What about `secondaryExcludeList`? 

It's hard to name this thing because it's only here to plan for when we have 2 different types of builds fully supported, one for the generic FPSci and the other for a purpose-built FPSci-based experiment. Calling it `experimentSpecific` makes it seem like you want that list when you're building an experiment, but I think we remove this exclude list for a specific experiment...

Reply:
Hmm, I think I may have misinterpreted it's use. If the intent is to exclude this content for "generic" builds I can now see the descriptive nature of the naming. Maybe we leave it as is.